lib/containers/home.js

```diff
@@ -32,7 +32,7 @@ export default Relay.createContainer(Home, {
   fragments: {
     home: () => Relay.QL`
       fragment on HomePage {
-        artwork_modules {
+        artwork_modules(max_rails: 50) {
```

Comment:
The current [max_rails](https://github.com/artsy/metaphysics/blob/master/schema/home/home_page_artwork_modules.js#L31) setup will grab more [generic genes](https://github.com/artsy/metaphysics/blob/master/schema/home/home_page_artwork_modules.js#L23) anyway, and [perhaps we'll make the api better in the future](https://github.com/artsy/metaphysics/issues/410).